src/Components/SignUp.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react'
+import { Form, Button, Container, Message, Header } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { registerUser } from '../reduxTokenAuthConfig'
+
+class SignUp extends Component {
+  state = {
+    email: '',
+    password: '',
+    password_confirmation: '',
+    level: '',
+    message: '',
+    errors_signup: '',
+    activeItem: '',
+  }
+
+  onChangeHandler = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value
+    })
+  }
+
+  handleChangeLevel = (e) => {
+    this.setState({ level: e.target.value, activeItem: e.target.value })
+  }
+
+  createUser = (e) => {
+    e.preventDefault();
+    const { history, registerUser } = this.props
+    const {
+      email,
+      password,
+      password_confirmation,
+      level,
+    } = this.state
+    registerUser({ email, password, password_confirmation, level })
+      .then(response => {
+        this.setState({ message: true })
+        setTimeout(function () { history.push('/') }, 3000)
```

Comment:
same comment as in Login component. maybe do this 1000 or 1500?

Reply:
Fixed!